src/aspects/aws-backup.ts

```diff
@@ -0,0 +1,27 @@
+import type { IAspect } from "aws-cdk-lib";
+import { CfnResource, TagManager } from "aws-cdk-lib";
+import { CfnTable } from "aws-cdk-lib/aws-dynamodb";
+import { CfnDBInstance } from "aws-cdk-lib/aws-rds";
+import type { IConstruct } from "constructs";
+
+/**
+ * Applies the tags that enable backups for supported resources.
+ *
+ * @see https://github.com/guardian/aws-backup
+ */
+export class AwsBackupTag implements IAspect {
+  /**
+   * These resources are backed up by https://github.com/guardian/aws-backup.
+   */
+  static resources: string[] = [CfnDBInstance.CFN_RESOURCE_TYPE_NAME, CfnTable.CFN_RESOURCE_TYPE_NAME];
```

Comment:
This is just `[ "AWS::RDS::DBInstance", "AWS::DynamoDB::Table" ]`. There's an argument that using the raw strings is more readable. Thoughts welcomed.

Reply:
Yeah, I find the strings a bit easier to follow.

In terms of `aws-backup` support:

* `AWS::DynamoDB::Table` has been well tested and we're [hoping to roll it out across P&E this quarter](https://docs.google.com/document/d/11EZtuVHCnjavE9AYLroiuDUsMzbp2ulFBtj4VkyCKNc/edit#heading=h.vwt7syo8ng40) (perhaps this will help inform whether defaulting to `true` would be a good choice or not?)
* `AWS::RDS::DBInstance` is being tested with Grafana, but we still have some questions for AWS before deciding on the exact approach / attempting a wider rollout
* `AWS::RDS::DBCluster` could also be supported in theory, but this is completely untested at the moment as we only have a [handful of clusters](https://metrics.gutools.co.uk/goto/ZesH_wVSk?orgId=1).

Reply:
Moved to strings (https://github.com/guardian/cdk/pull/2098/commits/32445bfe17b666f230b7291aece3718f7639e75e). It really was too fancy before 😁.